.github/workflows/version_bumper.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Sync repo files
+on:
+  schedule:
+    - cron: '21 14 * * *'
+jobs:
+  repo_sync:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    container:
+      image: quay.io/prometheus/golang-builder:base
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    strategy:
+      matrix:
+        include:
+          - repo: "prometheus/alertmanager"
+            role: "alertmanager"
+          - repo: "prometheus/blackbox_exporter"
+            role: "blackbox_exporter"
+          - repo: "prometheus/node_exporter"
+            role: "node_exporter"
+          - repo: "prometheus/prometheus"
+            role: "prometheus"
+          - repo: "prometheus/snmp_exporter"
+            role: "snmp_exporter"
```

Comment:
Can we do the matrix without hard-coding it? Like we do for the molecule tests

Reply:
I don't know if there's a sane way to do that, what if the repo is `prometheus-community/postgres_exporter` and the role is `postgres_exporter`?